pkg/cache/unavailableofferings.go

```diff
@@ -113,9 +113,9 @@ func (u *UnavailableOfferings) isFamilyUnavailable(sku *skewer.SKU, zone, capaci
 	return false
 }
 
-// MarkFamilyUnavailableWithTTL marks a VM family with custom TTL in a specific zone
+// MarkFamilyUnavailableAtCPUCount marks a VM family with custom TTL in a specific zone
```

Comment:
Minor: correct comment -- it should probably reference a cpuCount somehow.

Also maybe should reference that -1 is "all"